assets/src/scss/layout/_navbar.scss

```diff
@@ -208,6 +208,14 @@ a.nav-link {
   width: 100%;
   text-decoration: none !important;
 
+  span {
```

Comment:
I see we repeat this set of properties many times across the project, what about making them reusable? Perhaps in a different PR.